### PR TITLE
refactor: Use echo-jwt package

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: packages/authentication/jwt_auth.go [hunk 1]
```diff
@@ -161,87 +99,26 @@ func (j *JWTAuth) IssueJWT(username string, authClaims *WaspClaims) (string, err
...
-		}
-
-		return true
-	}
-
-	return jwtAuth, jwtAuthSkipper, jwtAuthAllow, nil
-}
-
 func AddJWTAuth(config JWTAuthConfiguration, privateKey []byte, userManager *users.UserManager, claimValidator ClaimValidator) (*JWTAuth, func() echo.MiddlewareFunc) {
+	const defaultJWTDuration = 24 * time.Hour
```
Comment (jorgemmsilva) [inline packages/authentication/jwt_auth.go]: should be a config param

**Comment by lmoe** [comment]
LGTM

There is a difference between the old and the new JWT payload:

old: `"aud": "wasp0"`
new: `"aud": [ "wasp0" ]`

Not sure important that is. It seems to work fine on the CLI.

**Review by lmoe — APPROVED** [review]

**Review by lmoe — CHANGES_REQUESTED** [review]
Sorry, I now tested a bit more.
It does not work with the CLI.

`chain list` seems to return an Unauthorized error result. I guess the JWT token is not read properly, not send properly, is invalid or incorrectly validated on the backend.

**Comment by howjmay** [comment]
> Sorry, I now tested a bit more. It does not work with the CLI.
> 
> `chain list` seems to return an Unauthorized error result. I guess the JWT token is not read properly, not send properly, is invalid or incorrectly validated on the backend.

Let me fix this

**Review by jorgemmsilva — APPROVED** [review]

**Comment by lmoe** [comment]
Tests are successful now, LGTM!

**Review by lmoe — APPROVED** [review]

**Pull request merged**